components/git/land.js

```diff
@@ -37,6 +37,11 @@ const landOptions = {
     describe: 'Assume "yes" as answer to all prompts and run ' +
     'non-interactively. If an undesirable situation occurs, such as a pull ' +
     'request or commit check fails, then git node land will abort.'
+  },
+  backport: {
+    describe: '',
```

Comment:
Needs a description, and likely needs some docs added to the README.md, or wherever the docs are.